main.tf

```diff
@@ -0,0 +1,11 @@
+# ----root/main.tf---
+
+module "network" {
+  source       = "./network"
+  vpc_cidr     = "10.0.0.0/16"
+  public_sn_count = 2
+  private_sn_count = 2
+  public_cidrs = [for i in range (2, 255, 2) : cidrsubnet("10.0.0.0/16", 8, i)]
+  private_cidrs = [for i in range (1, 255, 2) : cidrsubnet("10.0.0.0/16", 8, i)]
```

Comment:
It's better to replace `10.0.0.0/16` with a variable or a local.
So you don't have to change the value in multiple places if a subnet cider changes

Reply:
Also it might be a good idea to have newbits parameter of cidrsubnet function parametrized as well. So we can easily change subnet size if needed